src/Domain.LinnApps/Parts/Part.cs

```diff
@@ -4,7 +4,7 @@
 
     public class Part
     {
-        public int BridgeId { get; set; }
+        public long Id { get; set; }
```

Comment:
won't int be enough? I don't think we'll have a billion parts too soon

Reply:
int is definitely enough! I was playing around with hashing part numbers to make aId's (which ended up being very big numbers) and forgot to revert back to int. Will do now.